queries.sql

```diff
@@ -1,3 +1,26 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+/* Find all animals whose name ends in "mon". */
+SELECT * from animals WHERE like 'mon';
+
+/*List the name of all animals born between 2016 and 2019. */
+SELECT name from animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+
+/* List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT name from animals WHERE neutered = true and escape_attempts < 3;
+
+/*List the date of birth of all animals named either "Agumon" or "Pikachu". */
+SELECT date_of_birth FROM animals where (name='Agumon') OR (name='Pikachu');
+
+/* List name and escape attempts of animals that weigh more than 10.5kg */
+SELECT name,escape_attempts FROM animals where weight_kg > 10.5;
+
+/*Find all animals that are neutered.*/
+SELECT * from animals WHERE neutered = true;
+
+/*Find all animals not named Gabumon.*/
+SELECT * FROM animals WHERE name != 'Gabumon';
```

Comment:
- [ ] 👍 Excellent work on adding all 8 queries! However, to complete the requirements for the project, please ensure you provide screenshots for all of these queries. This will help reviewers and others better understand the results you've obtained from your queries.

Adding screenshots for each query will enhance the completeness of your PR documentation. Keep up the good work! 👍📸

Reply:
Thanks a lot, all the 8 queries have been now added to the PR description!